app/views/catalog/_collection_downloads.html.erb

```diff
@@ -1,16 +0,0 @@
-<% if collection_downloads.files.present? %>
```

Comment:
I'm a bit surprised to see this file be deleted.  The reason that this file is referenced as the `to_partial_path` of the (now) `DocumentDownloads` class is so that we can do this in a view (instead of putting this code explicitly in `_show_actions_box`):

```ruby
render Arclight::DocumentDownloads.new(document) 
```
(or perhaps even better having an accessor on the `SolrDocument` class that returns the `CollectionDownloads` instance for that document and then you can just `render document.download_links` or something along those lines)

This would have the benefit of making this code in the view a bit easier to read and refactor/move later if necessary.